From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Alexander Martinz <amartinz@shiftphones.com>
Date: Fri, 21 Apr 2023 14:40:48 +0500
Subject: [PATCH] (halium) adb: disable auth requirement for recovery by
 default

Our installer expects the recovery to have full adb access by default.

Change-Id: Ibf1a06fd9befb9fa379de591cf221db178f603fc
---
 daemon/main.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/daemon/main.cpp b/daemon/main.cpp
index 87e56dde..24d19524 100644
--- a/daemon/main.cpp
+++ b/daemon/main.cpp
@@ -225,7 +225,7 @@ int adbd_main() {
         // If we're on userdebug/eng or the device is unlocked, permit no-authentication.
         auth_required = android::base::GetBoolProperty("ro.adb.secure", false);
 #if defined(__ANDROID_RECOVERY__)
-        auth_required &= android::base::GetBoolProperty("ro.adb.secure.recovery", true);
+        auth_required &= android::base::GetBoolProperty("ro.adb.secure.recovery", false);
 #endif
     }
 #endif
